### PR TITLE
Fix amp validation

### PR DESCRIPTION
**Pull request by NataliaLKB**
The normal meta data was throwing AMP validation errors. Made it so that amp documents have slightly different meta data.

cc @johnduffell

### CODE REVIEW

File: common/app/views/fragments/metaData.scala.html [hunk 1]
```diff
@@ -8,22 +8,36 @@
...
-<link rel="dns-prefetch" href="@Configuration.ajax.url" />
-<link rel="dns-prefetch" href="@AnalyticsHost()" />
-<link rel="dns-prefetch" href="//j.ophan.co.uk" />
-<link rel="dns-prefetch" href="//ophan.theguardian.com" />
-<link rel="dns-prefetch" href="//oas.theguardian.com" />
-<link rel="dns-prefetch" href="@Configuration.debug.beaconUrl" />
+@if(amp) {
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+    <link rel="amphtml" href="/@item.id/amp">
```
Comment (NataliaLKB) [inline common/app/views/fragments/metaData.scala.html]: This tag is also throwing an error. @johnduffell do we still need it?

File: common/app/views/fragments/img.scala.html [hunk 2]
```diff
@@ -85,7 +85,6 @@
     @picture.largestImage.map { largestImage =>
         <amp-img src="@ImgSrc.getFallbackUrl(picture)"
         srcset="@ImgSrc.srcset(picture, widths)"
-        sizes="@widths.sizes"
```
Comment (NataliaLKB) [inline common/app/views/fragments/img.scala.html]: attr `sizes` is not allowed according to the validator.

File: common/app/views/fragments/metaData.scala.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@(item: model.MetaData)(implicit request: RequestHeader)
+@(item: model.MetaData, amp: Boolean)(implicit request: RequestHeader)
```
Comment (johnduffell) [inline common/app/views/fragments/metaData.scala.html]: you can actually set the default to false to save having to pass it in
`@(item: model.MetaData, amp: Boolean = false)(implicit request: RequestHeader)`

File: common/app/views/fragments/img.scala.html (hunk 2 above)
Comment (johnduffell) [inline common/app/views/fragments/img.scala.html]: we should raise an issue about that

File: common/app/views/fragments/metaData.scala.html (hunk 1 above)
Comment (johnduffell) [inline common/app/views/fragments/metaData.scala.html]: rememebr to wrap in an `if((item.isArticle && !item.isLiveBlog) || item.isGallery){` as well as moving it

**Comment by johnduffell** [comment]
great work :+1:

**Comment by NataliaLKB** [comment]
@johnduffell until we get a confirmation regarding the sizes attribute from the AMP team, lets keep it.

Would you mind doing another quick sanity check before I merge?

**Comment by johnduffell** [comment]
:+1: :speedboat:

**Pull request merged**